src/react/components/pages/editorPage/editorTagsInput.test.tsx

```diff
@@ -2,309 +2,37 @@ import { mount } from "enzyme";
 import React from "react";
 import MockFactory from "../../../../common/mockFactory";
 import { KeyCodes } from "../../../../common/utils";
-import EditorTagsInput, { IEditorTagsInputProps } from "./editorTagsInput";
+import EditorTagsInput from "./editorTagsInput";
+import { ITagsInputProps } from "vott-react";
 
 // tslint:disable-next-line:no-var-requires
-const TagColors = require("../../common/tagsInput/tagColors.json");
+const TagColors = require("vott-react/dist/lib/components/common/tagColors");
 
 describe("Tags Input Component", () => {
 
     const originalTags = MockFactory.createTestTags(15);
 
-    function createComponent(props: IEditorTagsInputProps) {
+    function createComponent(props: ITagsInputProps) {
         return mount(
-            <EditorTagsInput {...props}/>,
+            // Listing props one by one because of 'ref' typescript error.
+            // Example: https://github.com/ant-design/ant-design/issues/10405
+            <EditorTagsInput
+                tags={props.tags}
+                onChange={props.onChange}
+                placeHolder={props.placeHolder}
+                delimiters={props.delimiters}
+                onTagClick={props.onTagClick}
+                onShiftTagClick={props.onShiftTagClick}
+                onCtrlShiftTagClick={props.onCtrlShiftTagClick}
+            />,
         );
     }
 
-    it("tags are initialized correctly", () => {
```

Comment:
Most of these tests should still be in here. The modal tests can be removed because the logic for opening and closing the modal is now moved up a level (either `EditorFooter` or `ProjectForm`). But these other tests should still be valid.

Reply:
@tbarlow12 and @wbreza I just pushed changes that make all these tests pass. a number of them just needed displayHotKeys:true removed. I chose to leave it there commented out, because I think we should put that option back in, and it's nearly all there right now.